libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.cpp

```diff
@@ -144,6 +159,21 @@ const AP_Param::GroupInfo AC_AttitudeControl_Heli::var_info[] = {
     // @Increment: 1
     // @Units: Hz
     // @User: Standard
+
+    // @Param: RAT_PIT_SMAX
+    // @DisplayName: Pitch slew rate limit
```

Comment:
Let's change the display name to, "Pitch Rate Slew Limit" I think..

Reply:
@rmackay9 my comment is the same here as it is for yaw slew rate limit

Reply:
@rmackay9  this is limiting the rate of the command out to the actuator.  it is a slew rate limit not a pitch rate limit.  I deleted my comment for antenna tracker because that might command rate but I would still think that it should be called slew rate limiter even in that case.

Reply:
OK, as long as some thought has been put into it..